react/src/components/organisms/TabContainer/index.js

```diff
@@ -34,16 +63,36 @@ class TabContainer extends React.Component {
     const active = defaultTab;
     this.childrenWithProps = React.Children.map(children, (child, index) => {
       if (index === active) {
-        return React.cloneElement(child, { default: true });
+        return React.cloneElement(child, { default: true, a11yID: this.props.a11yID });
       }
       return child;
     });
+
     return(
       <TabContext.Provider value={this.state}>
         <div className={classes}>
-          <div className="ma__tab-container-head">{this.childrenWithProps}
-          </div>
-          {this.state.activeTab && <div className="ma__tab-container-body">{this.state.activeContent}</div>}
+          <span id={this.props.a11yID} className="ma__visually-hidden">Use left and right arrows to navigate between tabs, up and down arrows to navigate between active tab and its content.</span>
+          <ul className="ma__tab-container-head" role="tablist"
+              >{this.childrenWithProps}
+          </ul>
+
+
+
+// ADD onKeyDown to move the focus back to its paired tab. --> onKeyDown={this.handleTabContainerKeyDown}
+
+          {this.state.activeTab && <div className="ma__tab-container-body"
+                                    aria-labelledby={this.state.activeTab}
+                                    id={`tabpanel-${this.state.activeTab}`}
+                                    tabIndex="-1"
+                                    role="tabpanel"
+
+
+                                    onKeyDown={this.handleTabContainerKeyDown}
```

Comment:
This may take over keypresses for things within the tab container body...not sure this is a good place for setting this.